### ISSUE TITLE
Local and Federated tags missing.

### ISSUE THREAD

**Issue by r-clayton** [bookwyrm-social/bookwyrm#1273]
The Local and Federated tags have [disappeared from the Feed page](https://mastodon.sdf.org/@rclayton/106726739855067094). This happens on Fedora core 33 with chromium Version 91.0.4472.164 (Developer Build) Fedora Project (64-bit) and firefox 90.0 (64-bit) fedora-1.0.

Is the Discover page supposed to replace the Local and Federated tags?  If so, I vote for 1) much much better documentation and notification of this kind of change, 2) restoring the Local and Federated tags, and 3) a less eye-searing design for the Discover page (I don't know if these are separate issues because I don't know, other than the disappearing tags, what's going on).

**Comment by mouse-reeve** [comment]
It sounds like you feel blindsided by this change, and hurt that you weren't able to contribute to the thinking behind it. I want people who use BookWyrm to feel heard and involved in how the app progresses, so I appreciate you bringing these feelings up.

I solicited feedback on this change in a few places in advance of making it, including publicly on Mastodon, and there was a productive and fairly unanimous conversation about how to approach the activity timelines. It's tough for me to figure out how to make sure I'm including people, and while I did my best, I appreciate that it's an imperfect process, and that for you, it's led to frustrating and hurt.

When you have feedback or things aren't working for you, I need you to give constructive criticism rather than using language like "eye-searing." Receiving harsh language like that, without specifics I can use to improve, is hurtful -- it makes me feel like my work isn't valued. People who contribute to BookWyrm (myself included) are putting themselves out there and showing vulnerability by sharing their work, and it's important to me that this is a space that receives that vulnerability with care and compassion.
